fastcrawler_ui/routers/spiders.py

```diff
@@ -49,32 +54,45 @@ async def clients(
     return items
 
 
-@spider_router.post("/stop_task", status_code=status.HTTP_204_NO_CONTENT)
-async def stop_task(
-    task: TaskInput = Body(),
+def manage_tasks(
+    task_names: TasksInput,
+    crawler: FastCrawler,
+    spider_repository: SpiderRepository,
+    spider_controller: Type[SpiderController],
+    action: str,
```

Comment:
it seems there are only few possible options for action (either start or stop)
so you may be using Literal types, to support actions for this PR, although the intention of this PR was not this, but since you're touching this code it'd be awesome to fix.